terra-theme.config.js

```diff
@@ -0,0 +1,7 @@
+const themeConfig = {
+  exclude: [],
+  theme: '',
```

Comment:
The `exclude` and `theme` keys can be removed

Reply:
Changed here : https://github.com/cerner/terra-core/pull/2702/commits/172750bb3c24bc5288520d5238bba60b47b8687f